ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-http-client-source-interfaces.j2

```diff
@@ -0,0 +1,18 @@
+{% if ip_http_client_source_interfaces is defined and ip_http_client_source_interfaces is not none %}
+
+## IP HTTP Client Source Interfaces
+
+### IP HTTP Client Source Interfaces
+
+| VRF | Source Interface Name |
+| --- | --------------- |
+{%     for ip_http_client_source_interface in ip_http_client_source_interfaces %}
+{% if ip_http_client_source_interface['vrf'] is defined and ip_http_client_source_interface['vrf'] is not none %} {{ ip_http_client_source_interface['vrf'] }} {% else %}  default {% endif %}|{{ ip_http_client_source_interface['name'] }} |
```

Comment:
Please use ip_http_client_source_interface['vrf'] | arista.avd.default('default') instead of `if / then / else`

Reply:
Updated
```shell
| {{ ip_radius_source_interface['vrf'] | arista.avd.default('default') }} | {{ ip_radius_source_interface['name'] }} |
```